buildSrc/version.properties

```diff
@@ -1,5 +1,5 @@
 opensearch        = 3.0.0
-lucene            = 9.5.0-snapshot-d5cef1c
+lucene            = 9.5.0-snapshot-6700b7e
```

Comment:
The latest snapshots are needed to bring https://github.com/apache/lucene/pull/12016 in